### PR TITLE
Plugin for Eclipse IDE (over Broadway) as editor in Che

### PR DESCRIPTION
**Pull request by mickaelistria**
eclipse-ide

Signed-off-by: Mickael Istria <mistria@redhat.com>

### CODE REVIEW

**Comment by skabashnyuk** [comment]
how I can test the original image?
```
docker run -p 80:80  sopotc/eclipse-broadway
```

**Comment by slemeur** [comment]
@mickaelistria : I'm eager to see that :) !!

**Comment by benoitf** [comment]
I'm discussing with Mickaël, will handle it and add screencast

File: plugins/org.eclipse.che.editor.eclipseide/0.0.1/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+id: org.eclipse.che.editor.eclipseide
+version: 0.0.1
+type: Che Editor
+name: eclipse-ide
+title: Eclipse IDE (in browser using Broadway) as editor for Eclipse Che
+description: Eclipse IDE
+icon: https://github.com/eclipse/eclipse.platform/raw/master/platform/org.eclipse.sdk/eclipse256.png
+url: https://github.com/mickaelistria/che-editor-eclipseide/releases/download/untagged-267df4c0cbc3643d425b/che-editor-plugin.tar.gz
```
Comment (benoitf) [inline plugins/org.eclipse.che.editor.eclipseide/0.0.1/meta.yaml]: Hello, here is a proposed change
```suggestion
url: https://github.com/ws-skeleton/che-editor-eclipseide/releases/download/0.0.1/che-editor-plugin.tar.gz
```

Docker Image is updated to match openshift.io security (root access not allowed) (port number is also changed to be > 1024)
Also update Eclipse .tgz downloaded, squash layers to have smaller image than the original one

Update as well the meta.yaml to match new port number and avoid secured layer (else websocket of broadway is not working with this extra layer)

**Comment by benoitf** [comment]
For ppl wanted to try, here the plug-in attribute to use temporary
```
 "editor": "https://raw.githubusercontent.com/benoitf/custom-repo/master/plugins/org.eclipse.che.editor.eclipseide:0.0.1",
```

or here the full workspace.json https://gist.github.com/benoitf/4fda039549caa1374447c5dcfc2ab7bb

I've updated description of issue with a screencast

PS: as probe is missing, need to refresh the page when landing into Eclipse-IDE

File: plugins/org.eclipse.che.editor.eclipseide/0.0.1/meta.yaml (hunk 1 above)
Comment (mickaelistria) [reply plugins/org.eclipse.che.editor.eclipseide/0.0.1/meta.yaml]: ok!

**Comment by mickaelistria** [comment]
> I've updated description of issue with a screencast

Thanks a lot!

> PS: as probe is missing, need to refresh the page when landing into Eclipse-IDE

This is the kind of thing that we may try to fix later if it becomes useful, but at the moment, I suggest we apply the changes you recommend, create a small demo of it for interested parties to consider and leave it as it until there are requests.

**Comment by mickaelistria** [comment]
> are we good to merge?

I personally have no objection at all as I barely understand what I'm doing. I let you guys decide ;)

**Comment by skabashnyuk** [comment]
@benoitf WDYT?

**Pull request merged**